docs/user_guide/README.md

```diff
@@ -15,6 +15,12 @@ The task ```cli``` provided by the role, can also be directly implemented in you
 
 To manage multiple interfaces and vlans, the Network Engine role also offers [filter_plugins](https://github.com/ansible-network/network-engine/blob/devel/docs/plugins/filter_plugins.md) that turn lists of interfaces or vlans into ranges and vice versa.
```

Comment:
I realize this isn't part of your diffs, but it would warm my little techwriter heart if you could change vlans to VLANs in the readme.md :-)

Reply:
done :)